app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+
+class User
+	include DataMapper::Resource
+
+	attr_reader :password
+	attr_accessor :password_confirmation
+	validates_confirmation_of :password
+	validates_presence_of :email
+	validates_uniqueness_of(:email)
+
+	property :id, Serial
+	property :email, String, required: true
+	property :password_digest, Text
+
+	def password=(password)
+		@password = password
+		self.password_digest = BCrypt::Password.create(password)
+	end
+
+	def self.authenticate(email, password)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `password`. If it's necessary, use `_` or `_password` as an argument name to indicate that it won't be used.